server/Cargo.toml

```diff
@@ -47,7 +47,6 @@ log-panics = { version = "2.1.0", features = ["with-backtrace"] }
 
 # TODO: Remove. async_graphql is using 1.0 but SET_COOKIE was being imported from wrong version.
 http2 = { package = "http", version = "1.0.0" }
-log-panics = { version = "2.1.0", features = ["with-backtrace"] }
```

Comment:
Why was this removed? 👀

Reply:
Oh... it's above on line 46... maybe my base branch is out of date, one mo...

Reply:
I did see a commit in 2.3 that removed it 👀  was committed straight to 2.3?

Reply:
yeah... there's only one in 2.3 though, I'm confused..

Reply:
looks right now

Reply:
tests pass tho!